Runtime/JSContext.cs

```diff
@@ -15,10 +17,7 @@ namespace NodeApi;
 /// </remarks>
 public sealed class JSContext : IDisposable
 {
-    public JSContext()
-    {
-        JSNativeApi.Interop.Initialize();
-    }
+    private readonly napi_env _env;
```

Comment:
We can consider doing it in future. My current concern about subclassing `JSContext` from `JSValueScope` is that it could be used by `JSValue` to govern its lifetime, but it will be not correct. `JSContext` can survive across multiple calls as long as `napi_env` is not finalized. While the `JSValueScope` always must have a lifetime bound to current call stack. 
The only common part is that they both keep `napi_env`. It seems that a better option would be to have `JSValueScope` keeping a pointer to `JSContext` instead of `napi_value`.
We should also consider renaming `JSContext` to `JSModuleContext`.
I can do these changes in a follow up PR.